packages/typescript/README.md

```diff
@@ -74,6 +77,7 @@ To have the type definitions automatically found for imports, add `paths` in `ts
 | disableTypeCompilation        | boolean  | No       | false       | Disable compiling types for exposed components                                                                                                                                             |
 | disableDownloadingRemoteTypes | boolean  | No       | false       | Disable downloading types from remote apps                                                                                                                                                 |
 | typescriptFolderName          | string   | No       | `@mf-types` | The folder name to download remote types and output compiled types                                                                                                                         |
+| typescriptCompiledFolderName          | string   | No       | `_types` | The folder name to output the raw output from the ts compiler                                                                                                                         |
```

Comment:
I'm not convinced on having yet another `_types` folder nested inside `@mf-types` folder. The parent folder `@mf-types` pretty much serves the exact needs of having `_types` folder.